tools/functional-tester/etcd-tester/checks.go

```diff
@@ -15,6 +15,7 @@
 package main
 
 import (
+	"context"
```

Comment:
Nvm, we only run this with Go 1.7+, so should be ok to keep it as it is. Thanks.